src/utils/utils.ts

```diff
@@ -426,3 +426,36 @@ export const isValidProtocol = (protocolAddress: string): boolean => {
   );
   return validProtocolAddresses.includes(checkSumAddress);
 };
+
+/**
+ * Decode an encoded list of token ids
+ * @param encodedTokenIds encoded token ids from the criteria of an order
+ * @returns a list of token ids
+ */
+export const decodeTokenIds = (encodedTokenIds: string): string[] => {
```

Comment:
chatgpt generated

Reply:
nice. in the typedoc could you give a little explanation of how it is encoded to be decoded, so a succinct summary is understandable without reading into the code

Reply:
will do! planning on adding it in docs when we release